src/frame/js/actions/conversation.js

```diff
@@ -112,19 +113,19 @@ function postSendMessage(message) {
 
 function postUploadImage(message) {
     return (dispatch, getState) => {
-        const {config: {appId}, user: {_id}} = getState();
+        const {config: {appId}, user: {_id}} = getState();        
         const blob = getBlobFromDataUrl(message.mediaUrl);
-
         const data = new FormData();
+
         data.append('source', blob);
```

Comment:
Can we put `source` as the last object to be appended? It gives some flexibility to optimize validation server-side later on